js/modules/tilemap.src.js

```diff
@@ -387,6 +387,19 @@ seriesType('tilemap', 'heatmap'
      */
     tileShape: 'hexagon'
 }, {
+    // Use drawPoints method from old heatmap implementation
+    // Consider standarizing heatmap and tilemap into more consistent form.
+    drawPoints: function () {
+        var _this = this;
+        // In styled mode, use CSS, otherwise the fill used in the style
+        // sheet will take precedence over the fill attribute.
+        var func = this.chart.styledMode ? 'css' : 'animate';
```

Comment:
1. Do we need to declare the `func` variable? In other parts of HC we use it inline.
2. Wy do you use `css` instead of `attr` method?

Reply:
This method is only moved here the old heatmap implementation, in order to keep the way of how the tilemap has worked before, but agree. I removed that variable. Regarding `css` call, you can find the explanation in the comment above the code.